conda/cli/main_clean.py

```diff
@@ -33,14 +33,30 @@ def _get_size(*parts: str, warnings: Optional[List[Tuple[str, Exception]]] = Non
     return stat.st_size
 
 
-def _get_pkgs_dirs(pkg_sizes):
+def _get_pkgs_dirs(pkg_sizes: Dict[str, Dict[str, int]]) -> Dict[str, Tuple[str]]:
     return {pkgs_dir: tuple(pkgs) for pkgs_dir, pkgs in pkg_sizes.items()}
 
 
-def _get_total_size(pkg_sizes):
+def _get_total_size(pkg_sizes: Dict[str, Dict[str, int]]) -> int:
     return sum(sum(pkgs.values()) for pkgs in pkg_sizes.values())
 
 
+def _rm_rf(args, *parts: str, verbose: bool) -> None:
```

Comment:
Does it make sense to give `verbose` a default argument? I am saying this because it will make the code a little more clear. 

Right now, when you call this function, it's required that verbose be passed in as a keyword, but that's not immediately clear. A default value would help.

BUT, if this doesn't work because we want to enforce the caller actually pass in their own value, then that's fine too and we can leave this as-is.

Reply:
I agree, I disliked the blend of `*args` and `**kwargs` (not to mention actually having an `args` argument) so I've reworked that a little.

But yes, I opted to remove any default values bc there were no case where we weren't expecting the value to be passed in.

Reply:
Nice, I like that change!